<?xml version="1.0"?>

<launch>
  <arg name="file_config" default="$(find obstacle_stop)/config/detection_region_v2_3.yaml"/>

  <node pkg="obstacle_stop" type="obstacle_stop_v2_3.py" name="obstacle_stop_v2_3" output="screen">
    <param name="filepath" value="$(arg file_config)"/>
    <param name="speed_limit" value="0.5"/>
  </node>
</launch>
